docs/resources/Audit_Log.md

```diff
@@ -120,6 +121,10 @@ If no object is noted, there won't be a `changes` array in the entry, though oth
 | THREAD_UPDATE                         | 111   | Thread was updated                                        | [Thread](#DOCS_RESOURCES_CHANNEL/thread-metadata-object)                                                                                         |
 | THREAD_DELETE                         | 112   | Thread was deleted                                        | [Thread](#DOCS_RESOURCES_CHANNEL/thread-metadata-object)                                                                                         |
 | APPLICATION_COMMAND_PERMISSION_UPDATE | 121   | Permissions were updated for a command                    | [Command Permission](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-permissions-object-application-command-permissions-structure)\* |
+| AUTO_MODERATION_RULE_CREATE           | 140   | Auto Moderation rule was created                          | [Auto Moderation Rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object)                                                              |
+| AUTO_MODERATION_RULE_UPDATE           | 141   | Auto Moderation rule was updated                          | [Auto Moderation Rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object)                                                              |
+| AUTO_MODERATION_RULE_DELETE           | 142   | Auto Moderation rule was deleted                          | [Auto Moderation Rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object)                                                              |
+| AUTO_MODERATION_BLOCK_MESSAGE         | 143   | Message was blocked by AutoMod (according to a rule)      |                                                                                                                                                  |
```

Comment:
@hemu currently this audit log type's option contains the name of the rule which was triggered, are there any plans to provide the id of the rule as well?